subprojects/delivery/feature-toggles/src/main/kotlin/com/avito/android/plugin/SuspiciousTogglesCollector.kt

```diff
@@ -38,7 +33,7 @@ internal class SuspiciousTogglesCollector(
                     .find { it.codeLine.contains("defaultValue") }
 
                 if (lastChangeCodeItem == null) {
-                    logger.critical("Error: ${jsonToggle.key}", NoSuchElementException(jsonToggle.key))
+                    // TODO handle lastChangeCodeItem == null
```

Comment:
@eugene-krivobokov could we add a fail fast or we need a task to handle it properly?

Reply:
Let's just fail for now.

We need to delete this plugin. Unfortunately, it's broken and not worth fixing. See ON-122

Reply:
Is it used at Avito right now?

Reply:
The plugin is applied but tasks from in are not used anymore.
This is why we need to remove it from avito firstly and then remove it from github.